Index: olsrd-0.5.3/Makefile
===================================================================
--- olsrd-0.5.3.orig/Makefile	2007-08-05 01:31:21.000000000 +0200
+++ olsrd-0.5.3/Makefile	2007-08-05 01:32:13.000000000 +0200
@@ -180,6 +180,10 @@
 		$(MAKECMD) -C lib/quagga 
 		$(MAKECMD) -C lib/quagga DESTDIR=$(DESTDIR) install 
 
+txtinfo:
+		$(MAKECMD) -C lib/txtinfo clean
+		$(MAKECMD) -C lib/txtinfo 
+		$(MAKECMD) -C lib/txtinfo DESTDIR=$(DESTDIR) install 
 
 build_all:	cfgparser olsrd libs switch
 install_all:	install install_libs
