Index: olsrd-0.5.5/make/Makefile.linux
===================================================================
--- olsrd-0.5.5.orig/make/Makefile.linux	2008-02-08 09:39:17.000000000 +0000
+++ olsrd-0.5.5/make/Makefile.linux	2008-02-08 09:41:26.000000000 +0000
@@ -4,7 +4,7 @@
 
 ARCH := $(shell uname -m)
 DESTDIR ?=
-LIBDIR =	$(DESTDIR)$(shell if [ "$(ARCH)" = "x86_64" -a -d "/usr/lib64" ]; then echo "/usr/lib64"; else echo "/usr/lib"; fi)
+LIBDIR =	$(DESTDIR)/usr/lib
 
 SRCS += 	$(wildcard src/linux/*.c src/unix/*.c)
 HDRS +=		$(wildcard src/linux/*.h src/unix/*.h)
