--- olsrd-0.5.6-r6.orig/make/Makefile.kfbsd
+++ olsrd-0.5.6-r6/make/Makefile.kfbsd
@@ -3,11 +3,6 @@
 #

 PREFIX ?= /usr/local
-SBINDIR = $(PREFIX)/sbin
-ETCDIR = $(PREFIX)/etc
-LIBDIR = $(PREFIX)/lib
-DOCDIR = $(PREFIX)/share/doc
-MANDIR = $(PREFIX)/man

 SRCS +=                $(wildcard src/bsd/*.c) $(wildcard src/unix/*.c)
 HDRS +=                $(wildcard src/bsd/*.h) $(wildcard src/unix/*.h)
