diff --git a/lib/tas/Makefile b/lib/tas/Makefile
index 056353b..077918a 100644
--- a/lib/tas/Makefile
+++ b/lib/tas/Makefile
@@ -43,12 +43,13 @@ PLUGIN_VER =	0.1
 TOPDIR = ../..
 include $(TOPDIR)/Makefile.inc
 
-SRCS +=		$(wildcard src/lua/*.c)
-HDRS +=		$(wildcard src/lua/*.h)
+SRCS +=		
+HDRS +=		
 
 ifneq ($(OS),win32)
-LIBS +=		-lm
+LIBS +=		-lm -llua50
 endif
+CFLAGS += -I/usr/include/lua50
 
 ifneq ($(OS),$(filter $(OS),linux android))
 
